src/main/scala/is/hail/expr/Type.scala

```diff
@@ -563,6 +622,9 @@ abstract class TContainer extends Type {
   // FIXME LCM
   def contentsAlignment: Int = elementType.alignment.max(4)
 
+  override def unsafeOrdering(missingGreatest: Boolean): UnsafeOrdering =
+    TArray(elementType).unsafeOrdering(missingGreatest)
```

Comment:
OK, I'll have to move up all the elementOffset helper method stuff though. Will this play badly with the finality?